ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -143,13 +143,35 @@ private enum PluginSource {
 
   private static Dictionary<CelestialBody, Orbit> unmodified_orbits_;
 
+  private String bad_installation_popup_;
+
   public event Action render_windows;
 
   PrincipiaPluginAdapter() {
     // We create this directory here so we do not need to worry about cross-
     // platform problems in C++.
     System.IO.Directory.CreateDirectory("glog/Principia");
-    Log.InitGoogleLogging();
+    try {
+      Log.InitGoogleLogging();
+    } catch (DllNotFoundException e) {
+      UnityEngine.Debug.LogException(e);
+      string additional_information;
+      
+      if (File.Exists(Interface.kDllPath)) {
+        additional_information = "Please check that you downloaded the " +
+                                 "correct version of principia for your " +
+                                 "operating system, and that the right " +
+                                 "C++ ABI is installed in its appropriate " +
```

Comment:
C++ ABI is maybe not clear.  How about "Visual C++ Redistributable for Visual Studio 2015 if running on Windows or libwhateverc++ if running on Linux"?